pkg/cloudprovider/vsphere/route/config/consts_and_errors.go

```diff
@@ -39,4 +39,9 @@ const (
 
 	// DisplayNameMaxLength is the maximum length of static route display name
 	DisplayNameMaxLength = 255
+
+	// ProviderIDPrefix is the prefix of node's provider ID
+	ProviderIDPrefix = "vsphere://"
```

Comment:
We should probably consolidate this const with the other one defined so we only need to update in a single location should this ever change:
https://github.com/kubernetes/cloud-provider-vsphere/blob/master/pkg/cloudprovider/vsphere/util.go#L31

Would also be in favor of putting the bios uuid prefix in the same location

Reply:
Thanks for letting me know this, I should use `ProviderIDPrefix` from vsphere/util.go.
For `BiosUuidPrefix` actually it is a prefix from NSXT object, maybe I should put it in nsxt directory?